### PR TITLE
Fix: Invalid component name

### LINKED ISSUE #1533: Invalid component name
### What happened?

```
[Vue warn]: Invalid component name: "pages/rmrk/gallery/index.vue". Component names should conform to valid custom element name in html5 specification.
```

### Please reproduce in steps

Open any component name under `index.vue` or `_id.vue`

### Expected Behavior

to fix:

extend Component with `@Component` from ` 'nuxt-property-decorator'`
and pass the name inside

Example:
```ts
import { Component, Vue } from 'nuxt-property-decorator'

@Component({ name: 'GalleryPage' })
export default class GalleryPage extends Vue {
}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by codiini**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1533 
- [x] Fix Invalid component name issue

### Had issue bounty label ?
- [ ] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=<My_Kusama_Address>)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.

### CODE REVIEW

**Review by vikiival — CHANGES_REQUESTED** [review]
there is 6 other pages to process:

```bash
➜  nft-gallery git:(main-nuxt) ✗ find .  -regex ".*/_id.vue"         
./pages/rmrk/collection/_id.vue
./pages/rmrk/gallery/_id.vue
./pages/rmrk/u/_id.vue
./pages/rmrk/pack/_id.vue
./pages/rmrk/detail/_id.vue
./pages/index.vue
```

File: pages/rmrk/gallery/index.vue [hunk 1]
```diff
@@ -3,8 +3,8 @@
 </template>
 
 <script lang="ts" >
-import { Vue } from 'nuxt-property-decorator'
-
+import {Component, Vue } from 'nuxt-property-decorator'
```
Comment (vikiival) [inline pages/rmrk/gallery/index.vue:6]:
```suggestion
import { Component, Vue } from 'nuxt-property-decorator'
```

File: pages/rmrk/gallery/index.vue [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
+@Component({ name: 'GalleryPage' })
```
Comment (vikiival) [inline pages/rmrk/gallery/index.vue:7]:
```suggestion

@Component({ name: 'GalleryPage' })
```

**Review by vikiival — COMMENTED** [review]
Can you commit the suggestions? 🥺

File: pages/rmrk/gallery/index.vue (hunk 1 above)
Comment (roiLeo) [reply pages/rmrk/gallery/index.vue:6]: UwU

**Comment by codiini** [comment]
@vikiival Done!

**Review by vikiival — APPROVED** [review]
LGTM

**Comment by yangwao** [comment]
Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

**Pull request merged**